app/views/form-designer/completed-forms-email/change-email-address.html

```diff
@@ -0,0 +1,62 @@
+{% extends "layout-govuk-forms.html" %}
+
+{% set pageTitle = 'Are you sure you want to change this email address?' %}
+
+{% block pageTitle %}
+  {{ "Error:" if containsErrors }}{{pageTitle|safe}} - GOV.UK Forms
```

Comment:
I'm not 100% sure you need these `|safe` filters on pageTitle, since we know it'll be a plaintext string. But ignore this if you've tested it and do need it